lib/omnibus/software.rb

```diff
@@ -615,7 +615,7 @@ def with_standard_compiler_flags(env = {}, opts = {})
           {
             "LDFLAGS" => "-L#{install_dir}/embedded/lib #{arch_flag}",
             # If we're happy with these flags, enable SSE for other platforms running x86 too.
```

Comment:
If we're removing SSE, then this comment probably doesn't make sense any longer.